eZ/Publish/Core/Persistence/Cache/Tags/TagGenerator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Persistence\Cache\Tags;
```

Comment:
As mentioned during one of the internal demos old namespaces are deprecated. Since ibexa/core is undergoing rebranding right now it will cost much more time to provide BC layer. Instead you should define your class in `src/lib/Persistence/Cache/Tag` (not Tags, we rather use singular, except for test**s**) with:
```suggestion
namespace Ibexa\Core\Persistence\Cache\Tag;
```